homeassistant/components/recorder/core.py

```diff
@@ -418,12 +418,7 @@ def _reached_max_backlog_percentage(self, percentage: int) -> bool:
         # If they have more RAM available, keep filling the backlog
```

Comment:
Function doc string mentions returning a maximum, not a Boolean (line 412)

Reply:
docstring fixed in c8986aaba2cd5f52b1df860e3ed3b20c97aeb966